test/1729Essay.t.sol

```diff
@@ -1,49 +1,155 @@
 // SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.15;
+pragma solidity >=0.8.15;
 
 import "forge-std/Test.sol";
+import {IERC721Receiver} from
+    "openzeppelin-contracts/token/ERC721/IERC721Receiver.sol";
+
+import {OneSevenTwoNineEssay} from "../src/1729Essay.sol";
 import {Fleece} from "../src/Fleece.sol";
+import {Essay} from "../src/models/Essay.sol";
 
 import "../src/1729Essay.sol";
+import "openzeppelin-contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import "openzeppelin-contracts/token/ERC721/extensions/IERC721Metadata.sol";
+import "openzeppelin-contracts/token/ERC721/extensions/IERC721Enumerable.sol";
 
 contract OneSevenTwoNineEssayTest is Test {
     //
-    OneSevenTwoNineEssay essay;
+    using stdStorage for StdStorage;
+
+    OneSevenTwoNineEssay token;
+    address constant TESTAUTHOR = 0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045;
```

Comment:
Fine for now, we should rationalize between this and the writer1 etc. addresses below. Not blocking